teps/0025-hermekton.md

```diff
@@ -80,8 +82,14 @@ Tekton Pipelines will add support for a new "ExecutionMode" field on several obj
 That type will look like:
 
 ```go
+type HermeticMode string
+
+const (
+    NetworkingOff   HermeticMode = "networking-off"
+)
```

Comment:
I was thinking `enabled` and `disabled` for the string, but I don't feel strongly.

Reply:
agreed, updated the api - thank you!